extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -367,12 +365,6 @@ const transactionSubmissionSlice = createSlice({
     saveAssetSelectSource: (state, action) => {
       state.assetSelect.isSource = action.payload;
     },
-    showBlockedDomainWarning: (state) => {
```

Comment:
removed this stuff from redux to keep it w/in the component since it's only used there 😄